### PR TITLE
#1316 Link investment flow with public component

### PR DESCRIPTION
**Pull request by lizonp**

### CODE REVIEW

File: app/components/dashboard/eto-list/EtoList.tsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
+import { keyBy } from "lodash";
 import * as React from "react";
 import { Col } from "reactstrap";
 import { compose } from "redux";
 
+import { FormattedMessage } from "react-intl-phraseapp";
```
Comment (desfero) [inline app/components/dashboard/eto-list/EtoList.tsx]: `react-intl-phraseapp` should be in third-party imports

File: app/components/dashboard/eto-list/EtoList.tsx [hunk 2]
```diff
@@ -20,18 +22,38 @@ interface IDispatchProps {
...
-          <h4>{`${eto.company.brandName} - ${eto.company.name}`}</h4>
           {eto.state === "on_chain" && (
-            <Button onClick={() => startInvestmentFlow(eto)}>invest now</Button>
+            <EtoOverviewStatus
+              smartContractOnchain={eto.state === "on_chain"}
+              prospectusApproved={
+                !!keyBy(eto.documents, "documentType")["approved_prospectus"].name.length
+              }
+              termSheet={!!keyBy(eto.documents, "documentType")["termsheet_template"].name.length}
+              investmentAmount={"1234"} //TODO: connect proper one
```
Comment (desfero) [inline app/components/dashboard/eto-list/EtoList.tsx]: Do we have some story to connect the correct values? As for sure @SergejSoft will ask why the data are mocked :D

File: app/components/dashboard/eto-list/EtoList.tsx [hunk 3]
```diff
@@ -20,18 +22,38 @@ interface IDispatchProps {
...
+        <FormattedMessage id="dashboard.eto-opportunities" />
+      </SectionHeader>
     </Col>
     {etos.map((eto, index) => (
-      <Col xs={12} lg={6} key={index}>
-        {/* TODO: implement real ETO Card here */}
+      <Col xs={12} key={index}>
         <div className="mb-3">
-          <h4>{`${eto.company.brandName} - ${eto.company.name}`}</h4>
           {eto.state === "on_chain" && (
```
Comment (desfero) [inline app/components/dashboard/eto-list/EtoList.tsx]: use `EtoState.OnChain`

File: app/components/eto/shared/EtoPublicComponent.tsx [hunk 4]
```diff
@@ -120,17 +120,17 @@ export const EtoPublicComponent: React.SFC<IProps> = ({ companyData, etoData })
...
-          investmentAmount="100000"
-          status="campaigning"
-          raisedAmount="2000"
+          numberOfInvestors={12} //TODO: connect proper one
+          timeToClaim={12} //TODO: connect proper one
+          equityTokenPrice="10000" //TODO: connect proper one
+          investmentAmount="100000" //TODO: connect proper one
+          raisedAmount="2000" //TODO: connect proper one
+          newSharesGenerated={etoData.newSharesToIssue}
+          smartContractOnchain={etoData.state === "on_chain"}
```
Comment (desfero) [inline app/components/eto/shared/EtoPublicComponent.tsx]: compare with `EtoState.OnChain`

File: app/components/dashboard/eto-list/EtoList.tsx [hunk 5]
```diff
@@ -20,18 +22,38 @@ interface IDispatchProps {
...
     <Col xs={12}>
-      <SectionHeader>ETO Offerings</SectionHeader>
+      <SectionHeader>
+        <FormattedMessage id="dashboard.eto-opportunities" />
+      </SectionHeader>
     </Col>
     {etos.map((eto, index) => (
-      <Col xs={12} lg={6} key={index}>
-        {/* TODO: implement real ETO Card here */}
+      <Col xs={12} key={index}>
```
[the remark this answers is not in the document]
Comment (lizonp) [reply app/components/dashboard/eto-list/EtoList.tsx]: 👍

File: app/components/dashboard/eto-list/EtoList.tsx (hunk 1 above)
Comment (lizonp) [reply app/components/dashboard/eto-list/EtoList.tsx]: 👍

File: app/components/dashboard/eto-list/EtoList.tsx (hunk 2 above)
Comment (lizonp) [reply app/components/dashboard/eto-list/EtoList.tsx]: I need to dig into data we have, couldn't spot anythink that fits there so far. Part of them comes from bookbuilding I believe

File: app/components/dashboard/eto-list/EtoList.tsx (hunk 3 above)
Comment (lizonp) [reply app/components/dashboard/eto-list/EtoList.tsx]: 👍

File: app/components/eto/shared/EtoPublicComponent.tsx (hunk 4 above)
Comment (lizonp) [reply app/components/eto/shared/EtoPublicComponent.tsx]: 👍

**Pull request merged**